puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/deploy/website.groovy

```diff
@@ -0,0 +1,38 @@
+pipeline {
+    agent { label 'admin && sshkey' }
+
+    options {
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '3'))
+        disableConcurrentBuilds()
+        timestamps()
+    }
+
+    environment {
+        ruby_version = '2.5'
+        // Sync to the pivot-point on the web node
+        target_path = 'website@theforeman.org:rsync_cache/'
+    }
+
+    stages {
+        stage('Deploy website') {
+            steps {
+                git url: 'https://github.com/theforeman/theforeman.org'
+                try {
```

Comment:
I think this needs to go in a `script{ … }`, linter complains: `Expected a step @ line 21, column 17.'`